src/Tabs/Tabs.js

```diff
@@ -113,6 +114,29 @@ class Tabs extends Component {
     this.setState(newState);
   }
 
+  calculateTabWidthAndInkbarPosition(tabs, selectedIndex, defaultWidth) {
+    let i;
+    let left = 0;
+    let undefinedLabelWidths = 0;
+    for (i = 0; i < tabs.length; i++) {
+      const labelWidth = tabs[i].props.labelWidth;
+      if (labelWidth && i < selectedIndex)
+        left += labelWidth;
+      else if (!labelWidth)
+        undefinedLabelWidths++;
```

Comment:
A simple boolean is not enough. There are 3 possible scenarios in the `calculateTabWidthAndInkbarPosition`:
- Not a single Tab has defined `labelWidth` prop. This is the default behavior. Each tab is assigned a percentage of the container's width. If have 2 tabs, each gets 50%. The inkbar position is also a percentage (first tab would be 0% and the second would be 50%).
- Every single Tab has defined `labelWidth` prop. Now we have to use pixels instead of percentages. Each tab is assigned a string with the value of the `labelWidth` prop appending `px`. To calculate the inkbar position, we add the `labelWidth`s of every single component at the left of the selected tab.
- Some Tabs have defined `labelWidth` but other's have not. This is an error. The user probably forgot about one or more tabs.  It could be possible to fallback to using percentages for the tabs that are mssing the `labelWidth` prop, but calculating the inkbar position would be impossible because you can't add undefined values. In this scenario inkbar is not displayed.

I'm aware that this sounds a bit complicated, but I could not find easier alternatives. Adding numbers is far easier than parsing strings with percentages. 
I will move this function to a new module `TabUtils.js` and write a test for it before submitting a new PR to the `next` branch.

If you have any other suggestions or comments please let me know.